x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts

```diff
@@ -49,6 +54,7 @@ export enum AddInferencePipelineSteps {
 }
 
 const API_REQUEST_COMPLETE_STATUSES = [Status.SUCCESS, Status.ERROR];
+const DEFAULT_CONNECTOR_FIELDS = ['body', 'title', 'id', 'type', 'url'];
```

Comment:
This is probably a safe set of default fields for a connector... but also may not be 100% guaranteed. Do we still feel comfortable allowing a user to use these until there is a field mapping to prevent an error state?

Reply:
I'm good with this for now, since the target of the release and the story with it is a Connector story. If we have confirmed that the 8.5 ootb connectors use these fields (I think crawler uses ` body_content`?) then no objection to leaving as-is!

Reply:
crawler's get a default field mapping on creation. So they will never hit this case. So this array will only be used for an index that pass the `isConnectorIndex` guard (checks for .connector key) and has an empty field mapping.

Which essentially means only an API-based index with an empty field mapping will see the error I'm adding in this PR.

Reply:
Wait, are we supporting API-based indices in 8.5? I thought we weren't 🙈

Reply:
We are, we've added copy to the UI to called out the user's need to use the pipeline explicitly. See #142406